fetch.bs

```diff
@@ -4482,6 +4492,11 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
  <li><p>If <i>authentication-fetch flag</i> is set, then create an <a>authentication entry</a>
  for <var>request</var> and the given realm.
 
+ <li><p>If <i>revalidatingFlag</i> is set and <var>httpRequest</var>'s <a for=request>cache mode</a>
+ is "<code>async-stale-revalidated</code>" then asynchronously perform a
```

Comment:
This needs a comma before then. "asynchronously" is not a thing, perhaps "in parallel"?